### ISSUE TITLE
Cybran Land Scout Deselection

### ISSUE THREAD

**Issue by ComradeStryker** [FAForever/fa#4891]
**Describe the feature**
The Seraphim Selen (T1 Land Scout) has a cloaking ability. 
And when this ability is active, the unit is unable to be selected (with a large group of other units,) to prevent it from moving.
However, the Cybran Mole, which also has the same ability, does not have the same feature.

**Expected behavior**
Cybran Mole to have a Deselection feature, same as the Seraphim Combat Scout has.

**Screenshots**
Selecting all units on screen:
![Screenshot 2023-04-29 085027](https://user-images.githubusercontent.com/121060254/235311599-3ad88037-a927-45af-bcae-057d326ea17f.png)

Result: Selen and Mole are both Cloaked, yet Mole isn't deselected.
![Screenshot 2023-04-29 085038](https://user-images.githubusercontent.com/121060254/235311602-12add1b0-6f4b-4152-9b2e-16ad7418e526.png)

**Additional context**
Holding Shift (or not) when selecting a Selen, regardless, will always be deselected if another unit is selected with it.

**Comment by Garanas** [comment]
Question is whether this is a bug or a feature? What are the thoughts about this?

**Comment by ComradeStryker** [comment]
I thought this was a feature, as the deselection only happens when the Selen is cloaked.

**Comment by Garanas** [comment]
Yes, that is an intentional feature. But is it intentional that the Mole is left out?

**Comment by ComradeStryker** [comment]
Wouldn't make sense why it should've been.
They both share the exact same role and abilities.
Would make sense (to new players. as well) if units were concise.

**Comment by lL1l1** [comment]
> However, the Cybran T1 Land Scout (Mole), which also has the same ability [as the selen], does not have the same [deselection] feature.

Mole's cloak is distinctly different in that it is mobile. You can cloak a mole, bring it with your raid, and if the enemy doesn't have a scout the mole will live due to its cloak and gather useful intel. You would have to fight the UI if you wanted a cloaked mole + mantis to attack together.
Moving a cloaked mole costs nothing, while moving a cloaked Selen costs you 3 seconds of re-cloak time, which is why Selen gets automatically deselected.
